File: FbxCppTests/FbxTimeCodeTest.cpp

#include "Tests.h"

using namespace std;

void FbxTimeCode_Constants()
{
    // expect:
    AssertEqual(141120L, FBXSDK_TC_MILLISECOND);
    AssertEqual(141120000L, FBXSDK_TC_SECOND);
    AssertEqual(46186158L, FBXSDK_TC_LEGACY_MILLISECOND);
    AssertEqual(46186158000L, FBXSDK_TC_LEGACY_SECOND);
}

void FbxTimeCodeTest::RegisterTestCases()
{
    AddTestCase(FbxTimeCode_Constants);
}

File: FbxSharpTests/FbxTimeCodeTest.cs
using System;
using NUnit.Framework;
using FbxSharp;

namespace FbxSharpTests
{
    [TestFixture]
    public class FbxTimeCodeTest : TestBase
    {
        [Test]
        public void FbxTimeCode_Constants()
        {
            // expect:
            Assert.That(FbxTimeCode.FBXSDK_TC_MILLISECOND, Is.EqualTo(141120L));
            Assert.That(FbxTimeCode.FBXSDK_TC_SECOND, Is.EqualTo(141120000L));
            Assert.That(FbxTimeCode.FBXSDK_TC_LEGACY_MILLISECOND, Is.EqualTo(46186158L));
            Assert.That(FbxTimeCode.FBXSDK_TC_LEGACY_SECOND, Is.EqualTo(46186158000L));
        }
    }
}
